chromium_src/content/public/common/content_features.cc

```diff
@@ -23,7 +23,6 @@ OVERRIDE_FEATURE_DEFAULT_STATES({{
     {kDigitalGoodsApi, base::FEATURE_DISABLED_BY_DEFAULT},
     {kDIPS, base::FEATURE_DISABLED_BY_DEFAULT},
     {kFedCm, base::FEATURE_DISABLED_BY_DEFAULT},
-    {kFirstPartySets, base::FEATURE_DISABLED_BY_DEFAULT},
```

Comment:
Seems like we'd want to have the affected code disabled in another way then.

Reply:
Yeah, that's not right. @samartnik do you happen to know what commit those files belong with?

Reply:
cc: @ShivanKaul

Reply:
@ShivanKaul wrote:

> The new name for First Party Sets is Related Website Sets. I think as long as we make the following function return false, it will be equivalent to the flag being off: [PrivacySandboxSettings::AreRelatedWebsiteSetsEnabled()](https://source.chromium.org/chromium/chromium/src/+/main:components/privacy_sandbox/privacy_sandbox_settings.h;l=332;drc=805c9ee2d426bd2ef36ec6128553bed917f1b5e4;bpv=1;bpt=1)

We provide a stub implementation of `PrivacySandboxSettings` that returns `false` from that function here:
https://github.com/brave/brave-core/blob/master/components/privacy_sandbox/brave_privacy_sandbox_settings.cc#L224-L226